components/MovieCard/index.js

```diff
@@ -6,6 +6,7 @@ const API_IMG = "https://image.tmdb.org/t/p/w500/";
 
 export default function MovieCard() {
   const [movies, setMovies] = useState([]);
+  const [activeItem, setActiveItem] = useState(undefined);
```

Comment:
This is personal preference, but I'd initialize the state with a `-1` here because we're dealing with indices and `-1` is conventionally used to denote an undefined state. It's like the `findIndex` or `indexOf` array methods.

Reply:
Thank you for your comment, I will implement this in the next pull :)